NDB.Covid19/NDB.Covid19.iOS/appcenter-pre-build.sh

```diff
@@ -91,6 +91,12 @@ then
     exit
 fi
 
+if [ -z "APPCENTER_DIAGNOSTICS_TOKEN" ]
+then
+    echo "You need define the APPCENTER_DIAGNOSTICS_TOKEN variable in App Center"
+    exit
```

Comment:
Since this token is supported only for Alpha/Beta builds, I would not put an exit here, so production script can continue when this token is missing